cartographer/mapping/sparse_pose_graph.cc

```diff
@@ -84,22 +95,39 @@ proto::SparsePoseGraph SparsePoseGraph::ToProto() {
     Eigen::Map<Eigen::Matrix<double, 6, 6>>(
         constraint_proto->mutable_sqrt_lambda()->mutable_data()) =
         constraint.pose.sqrt_Lambda_ij;
-    // TODO(whess): Support multi-trajectory.
+
     constraint_proto->mutable_submap_id()->set_submap_index(constraint.i);
+    constraint_proto->mutable_submap_id()->set_trajectory_id(
+        trajectory_indices[trajectory_nodes[constraint.i]
+                               .constant_data->trajectory]);
+
     constraint_proto->mutable_scan_id()->set_scan_index(constraint.j);
+    constraint_proto->mutable_scan_id()->set_trajectory_id(
+        trajectory_indices[trajectory_nodes[constraint.j]
+                               .constant_data->trajectory]);
     constraint_proto->set_tag(mapping::ToProto(constraint.tag));
   }
 
-  // TODO(whess): Support multi-trajectory.
-  proto::Trajectory* const trajectory = proto.add_trajectory();
-  *trajectory = mapping::ToProto(GetTrajectoryNodes());
```

Comment:
I think this `mapping::ToProto()` function is not used anywhere else. Should it be removed since it only supports a single trajectory and should not be used in the future?